lib/std/os/posix_spawn.zig

```diff
@@ -46,9 +46,13 @@ const posix_spawn = if (builtin.target.isDarwin()) struct {
             }
         }
 
-        pub fn deinit(self: *Attr) void {
-            system.posix_spawnattr_destroy(&self.attr);
-            self.* = undefined;
+        pub fn deinit(self: *Attr) Error!void {
+            defer self.* = undefined;
+            switch (errno(system.posix_spawnattr_destroy(&self.attr))) {
+                .SUCCESS => {},
+                .INVAL => unreachable,
+                else => |err| return unexpectedErrno(err),
+            }
```

Comment:
If this is not expected to return any error then it'd be better to just make the else prong `unreachable`, see `std.os.munmap` for an example of this.